data/modules/whatshap/Sample_3_3p.phased.tsv

```diff
@@ -0,0 +1 @@
+#sample	chromosome	file_name	variants	phased	unphased	singletons	blocks	variant_per_block_median	variant_per_block_avg	variant_per_block_min	variant_per_block_max	variant_per_block_sum	bp_per_block_median	bp_per_block_avg	bp_per_block_min	bp_per_block_max	bp_per_block_sum	heterozygous_variants	heterozygous_snvs	phased_snvs	block_n50
```

Comment:
This file appeared to be empty - is that intentional?

Reply:
Yes, this is the file WhatsHap outputs when it can find no phased variants. The module expands this to have 0 values in all fields, so the statistics can still be used for the HTML report.